libs/libc/misc/lib_pathbuffer.c

```diff
@@ -87,7 +87,7 @@ FAR char *lib_get_pathbuffer(void)
   /* Try to find a free buffer */
 
   nxmutex_lock(&g_pathbuffer.lock);
-  index = ffs(g_pathbuffer.free_bitmap) - 1;
+  index = ffsl(g_pathbuffer.free_bitmap) - 1;
```

Comment:
change line 45 to `unsigned long free_bitmap;`

Reply:
Done